src/components/[guild]/Requirements/components/RelativeDataBlockDate.tsx

```diff
@@ -0,0 +1,12 @@
+import formatRelativeTimeFromNow from "utils/formatRelativeTimeFromNow"
+import DataBlock from "./DataBlock"
+
+type Props = {
+  data: number
+}
+
+const RelativeDataBlockDate = ({ data }: Props): JSX.Element => (
```

Comment:
The component name could be `DataBlockWithRelativeDate`, so we use consistent names across our `DataBlock...` components